storage/local/storage.go

```diff
@@ -87,6 +87,11 @@ var (
 		"The maximum number of chunks that can be waiting for persistence before sample ingestion will stop.",
 		nil, nil,
 	)
+	maxMemChunksDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, subsystem, "max_memory_chunks"),
+		"The maximum number of chunks that can be holding in the memory",
```

Comment:
The configured maximum number of chunks that can be held in memory

It can go a little above this.